client/app/telecom/overTheBox/details/overTheBox-details.html

```diff
@@ -1,45 +1,36 @@
 <div class="overthebox">
-    <header class="text-right">
-        <button class="btn btn-primary"
-                type="button"
-                data-ng-click="OverTheBoxDetails.LaunchAction('reboot')"
-                data-ng-disabled="!OverTheBoxDetails.availableAction.reboot"
-                data-translate="overTheBox_action_reboot_label">
-        </button>
-        <div class="btn-group"
-             data-uib-dropdown
-             data-dropdown-append-to-body>
+    <header>
+        <oui-dropdown class="d-inline-block" align="start">
```

Comment:
You probably could use: `<oui-action-menu>` (ref: https://master.ui-kit.ovh/#!/oui-angular/action-menu).

Reply:
Yes I could use it, but this will make a change compare to what is currently displayed to the user. 
The small text would been set up in translation.